libcudacxx/include/cuda/std/detail/libcxx/include/cstddef

```diff
@@ -57,7 +57,7 @@ using ::ptrdiff_t;
 using ::size_t;
 
 #if defined(__CLANG_MAX_ALIGN_T_DEFINED) || defined(_GCC_MAX_ALIGN_T) || defined(__DEFINED_max_align_t) \
-  || defined(__NetBSD__)
+  || defined(__NetBS)
```

Comment:
Do we just want to drop this alltogether?

Reply:
You mean the entire ifdef structure or just the netbsd part?

Reply:
Just the NetBSD stuff